docs/ml-guide.md

```diff
@@ -207,7 +207,7 @@ val model1 = lr.fit(training.toDF)
 // we can view the parameters it used during fit().
 // This prints the parameter (name: value) pairs, where names are unique IDs for this
 // LogisticRegression instance.
-println("Model 1 was fit using parameters: " + model1.fittingParamMap)
+println("Model 1 was fit using parameters: " + model1.extractParamMap)
```

Comment:
I should have noticed before: The right way to access the parameters is really: ```model1.parent.extractParamMap``` since that gets the Params for the parent Estimator, which could potentially differ from the Model Params.  Could you please update these 4 examples?  Other than this, the changes look fine.  Thanks!